airbyte-integrations/connectors/source-paypal-transaction/source_paypal_transaction/schemas/transactions.json

```diff
@@ -0,0 +1,250 @@
+        "name": {
+          "type": "string"
+        },
+        "address": {
+          "type": "object",
+          "properties": {
+            "line1": {
+              "type": "string"
+            },
+            "line2": {
+              "type": "string"
+            },
+            "city": {
+              "type": "string"
+            },
+            "country_code": {
+              "type": "string"
+            },
+            "postal_code": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "cart_info": {
+      "type": "object",
+      "properties": {
+        "item_details": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "item_code": {
+                "type": "string"
+              },
+              "item_name": {
+                "type": "string"
+              },
+              "item_description": {
+                "type": "string"
+              },
+              "item_quantity": {
+                "type": "string"
+              },
+              "item_unit_price": {
+                "type": "object",
+                "properties": {
+                  "currency_code": {
+                    "type": "string"
+                  },
+                  "value": {
+                    "type": "string"
+                  }
+                }
+              },
+              "item_amount": {
+                "type": "object",
+                "properties": {
+                  "currency_code": {
+                    "type": "string"
+                  },
+                  "value": {
+                    "type": "string"
+                  }
+                }
+              },
+              "tax_amounts": {
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "tax_amount": {
+                      "type": "object",
+                      "properties": {
+                        "currency_code": {
+                          "type": "string"
+                        },
+                        "value": {
+                          "type": "string"
+                        }
+                      }
+                    }
+                  }
+                }
+              },
+              "total_item_amount": {
+                "type": "object",
+                "properties": {
+                  "currency_code": {
+                    "type": "string"
+                  },
+                  "value": {
+                    "type": "string"
+                  }
+                }
+              },
+              "invoice_number": {
+                "type": "string"
+              }
+            }
+          }
+        }
+      }
+    },
+    "store_info": {
+      "type": "object"
+    },
+    "auction_info": {
+      "type": "object"
+    },
+    "incentive_info": {
+      "type": "object"
+    }
```

Comment:
The above three objects do not have `properties`. Based on the [API doc](https://developer.paypal.com/docs/api/transaction-search/v1/), they should be:

```json
"store_info": {
  "type": "object",
  "properties": {
    "store_id": {
      "type": ["null", "string"]
    },
    "terminal_id": {
      "type": ["null", "string"]
    }
  }
},
"auction_info": {
  "type": "object",
  "properties": {
    "auction_site": {
      "type": ["null", "string"]
    },
    "auction_item_site": {
      "type": ["null", "string"]
    },
    "auction_buyer_id": {
      "type": ["null", "string"]
    },
    "auction_closing_date": {
      "type": ["null", "string"]
    }
  }
},
"incentive_info": {
  "type": "object",
  "properties": {
    "incentive_details": {
      "type": "array",
      "items": {
        "$ref": "#/$defs/incentive_detail"
      }
    }
  },
  "$defs": {
    "incentive_detail": {
      "type": "object",
      "properties": {
        "incentive_type": {
          "type": ["null", "string"]
        },
        "incentive_code": {
          "type": ["null", "string"]
        },
        "incentive_amount": {
          "type": "object",
          "properties": {
            "currency_code": {
              "type": "string"
            },
            "value": {
              "type": "string"
            }
          }
        },
        "incentive_program_code": {
          "type": ["null", "string"]
        }
      }
    }
  }
}
```

The `currency_code` and `value` properties inside `incentive_amount` object are required. All the other fields are nullable (I think).

Reply:
thanks!